lib/cell.rb

```diff
@@ -25,4 +25,11 @@ def fire_upon
     @ship.hit
   end
 
+  def render(reveal = false)
+    return 'S' if reveal && !empty? && !fired_upon?
+    return '.' if fired_upon? == false
+    return 'H' if fired_upon? == true && !empty? && !@ship.sunk?
+    return 'X' if @ship.sunk? == true 
```

Comment:
Looks good, we'll refactor for redundancy on some of the methods after we submit iteration 3 and go for a refactor.